playwright_tests/Toaster.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+
+const TIMEOUT_MESSAGE = 'Changes successfully saved.';
+const NULL_TIMEOUT_MESSAGE = 'Uploading videos.';
+
+test('toast should appear after a button click', async ({ page }) => {
+	await page.goto('/Toaster');
+	await page.getByRole('button', { name: 'Delete' }).click();
+	await page.waitForTimeout(1000);
+	const isToastPresent = await page.$(`text="${TIMEOUT_MESSAGE}"`);
+	expect(isToastPresent).not.toBeNull();
+});
+
+test('toast should disappear after a button click', async ({ page }) => {
+	await page.goto('/Toaster');
+	await page.getByRole('button', { name: 'Delete' }).click();
+	await page.waitForTimeout(5000);
+	const isToastPresent = await page.$(`text="${TIMEOUT_MESSAGE}"`);
+	expect(isToastPresent).toBe(null);
+});
+
+test('toast should not disappear if timeout is set to null', async ({
+	page
+}) => {
+	await page.goto('/Toaster');
+	await page.getByRole('button', { name: 'Upload' }).click();
+	await page.getByText(`text="${NULL_TIMEOUT_MESSAGE}"`);
```

Comment:
this doesn't do anything as is. `page.getByText` returns a `Locator` that can then be used to either interact with or run expectations against, but on its own it doesn't do anything. To assert that this text is in the page, you might do something like:
```ts
await expect(page.getByText("Uploading videos")).toBeVisible();
```